src/cljc/athens/common_events/resolver/undo.cljc

```diff
@@ -40,6 +40,38 @@
     (vec (concat repr-ops save-ops))))
 
 
+(defmethod resolve-atomic-op-to-undo-ops :block/move
+  [_ evt-db {:op/keys [args]}]
+  (println "resolve atomic op to undo ops args -->" args)
+  (let [{:block/keys [uid]}       args
+        {:block/keys [order]}     (common-db/get-block evt-db [:block/uid uid])
+        ;; How to get the relative position of a block?
+        ;; - If the block has 0 block-order then we can say it is children of some block
+        ;;   hence we can reference its position via parent
+        ;; - If the block has order > 0 then we know it atleast a previous sibling so we
+        ;;   can use this previous block to reference the position
+        parent                    (common-db/get-parent evt-db [:block/uid uid])
+        parent-page?              (if (and (= 0 order)
+                                           (:node/title parent))
+                                    true
+                                    false)
+        parent-ref                (if parent-page?
+                                    (:node/title parent)
+                                    (:block/uid parent))
+        {prev-sib-uid :block/uid} (common-db/nth-sibling evt-db uid -1)
+        prev-block-pos            (cond
+                                    (= order 0) :parent
+                                    :else       :prev-sib)
+        position                  (cond
```

Comment:
Can you use the new `common-db/get-position` here instead? It was merged into the undo branch yesterday.